src/Login/LoginCollection.php

```diff
@@ -0,0 +1,128 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Login;
+
+class LoginCollection implements \IteratorAggregate
+{
+    private array $failedLoginAttempts;
+
+    private array $apiLogins;
+
+    private array $adminLogins;
+
+    private array $adminLoginsPerDay;
+
+    private array $newAccounts;
+
+    public function getIterator(): \ArrayIterator
+    {
+        return new \ArrayIterator();
+    }
```

Comment:
you never use an object of this class with a `foreach`, why does it implement iterator aggregate?